{{cookiecutter.repo_name}}/setup.py

```diff
@@ -31,7 +31,7 @@
 ]
 
 tests_require = [
-    'WebTest >= 1.3.1',  # py3 compat
```

Comment:
comments when a dep is pinned are really useful, just because we are dropping py2 does not mean that it makes sense to stop explaining why we want at least this version of webob

Reply:
Normally I would agree, but no one will be using Python 2 for this cookiecutter, and that version of webtest was released on Sep 23, 2011. Let's go all in to Python 2!

Reply:
I don't think I was really clear before, your best options are to 1) drop the version pin or to 2) restore the comment. Leaving a dependency pinned without a reason why it's pinned is strictly worse than pinning with a comment. I realize pytest is also pinned without a comment and that should be remedied as well, as I'm looking at it and have no clue why it's there.

Reply:
I dropped the pins. Fixed in 1ae840166e6ba28d9f5d5471022cab0964ebcbd1.

There was a [bug in pytest 3.7.3](https://github.com/pytest-dev/pytest/issues/3506#issuecomment-416105809), which has since been fixed. I'll add a comment when pinning in the future.